### DISCUSSION TITLE
Help toggling settings

### DISCUSSION THREAD

**Discussion by oredaze** [gokcehan/lf#1192] [Q&A] [answered]
I am trying to get a ranger-like preview toggle. Basically removing the preview column without changing layout. It is achieved by disabling the preview column and setting ratios as per below.
This is where I am at, but I don't know how to check for the status of the preview setting and I don't know how to apply 2 settings at once.
```
cmd preview_toggle %{{
  if [ $lf_preview "on" ]; then
    lf -remote "send $id set preview!"
    lf -remote "send $id set ratios 1:5"
  else
    lf -remote "send $id set preview!"
    lf -remote "send $id set ratios 1:2:3"
  fi
}}
map zp preview_toggle
```

**Comment by joelim-work** [comment, marked as answer]
To check whether the preview column is enabled you need to use `=` to compare the strings. Also the string you should compare to is `true`, not `on`.

To apply two settings at once, you need to separate them using `;`, and also add `:` at the start (this means an `lf` command, and is normally implied but needs to be specified if chaining multiple commands using `;`).

Here is the modified config:

```
cmd preview_toggle %{{
  if [ "$lf_preview" = "true" ]; then
    lf -remote "send $id :set preview!; set ratios 1:5"
  else
    lf -remote "send $id :set preview!; set ratios 1:2:3"
  fi
}}
map zp preview_toggle
```

**Reply by joelim-work** [reply]
@oredaze Thanks for providing this configuration example. I have added it to the [wiki](https://github.com/gokcehan/lf/wiki/Tips#toggling-the-preview-column) so that other users can benefit from this.

**Reply by oredaze** [reply]
Thank you so much.
